cmd/neofs-adm/internal/modules/morph/policy.go

```diff
@@ -0,0 +1,64 @@
+package morph
+
+import (
+	"fmt"
+	"math/big"
+	"strconv"
+	"strings"
+
+	"github.com/nspcc-dev/neo-go/pkg/core/native/nativenames"
+	"github.com/nspcc-dev/neo-go/pkg/io"
+	"github.com/nspcc-dev/neo-go/pkg/smartcontract/callflag"
+	"github.com/nspcc-dev/neo-go/pkg/vm/emit"
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+)
+
+const (
+	execFeeParam      = "ExecFeeFactor"
+	storagePriceParam = "StoragePrice"
+	setFeeParam       = "FeePerByte"
+)
+
+func setPolicyCmd(cmd *cobra.Command, args []string) error {
+	wCtx, err := newInitializeContext(cmd, viper.GetViper())
+	if err != nil {
+		return fmt.Errorf("can't to initialize context: %w", err)
+	}
+
+	policyHash, err := wCtx.Client.GetNativeContractHash(nativenames.Policy)
+	if err != nil {
+		return fmt.Errorf("can't get policy contract hash: %w", err)
+	}
+
+	bw := io.NewBufBinWriter()
+	for i := range args {
+		kv := strings.SplitN(args[i], "=", 2)
+		if len(kv) != 2 {
+			return fmt.Errorf("invalid parameter format, must be Parameter=Value")
+		}
+
+		switch kv[0] {
+		case execFeeParam, storagePriceParam, setFeeParam:
+		default:
+			return fmt.Errorf("parameter must be one of %s, %s and %s", execFeeParam, storagePriceParam, setFeeParam)
+		}
+
+		value, err := strconv.ParseUint(kv[1], 10, 32)
+		if err != nil {
+			return fmt.Errorf("can't parse parameter value '%s': %w", args[1], err)
+		}
+
+		emit.AppCall(bw.BinWriter, policyHash, "set"+kv[0], callflag.All, big.NewInt(int64(value)))
```

Comment:
does it work ok with small `value`? as I remember small numbers has its own opcodes. or it is ok to write any number as `big.Int`?

Reply:
`big.Int` will adjust it's behaviour appropriately. But we still can use `int64` here, `val` is 32-bit.